src/anders_mathiesen_ex/ex02/file_stats.py

```diff
@@ -0,0 +1,26 @@
+def char_counts(textfilename):
+    result_list = [0 for _ in range(256)]
+    with open(textfilename, 'r', encoding='utf8') as fp:
+        for line in fp.readlines():
```

Comment:
you can avoid this for-loop by using `.read()` instead, which should read the entire file into a single string. You can then iterate through that.